internal/lintcontext/create_contexts.go

```diff
@@ -36,6 +37,16 @@ func CreateContexts(filesOrDirs ...string) ([]LintContext, error) {
 			continue
 		}
 
+		if strings.HasSuffix(strings.ToLower(fileOrDir), ".tgz") {
+			ctx := new()
+			if err := ctx.loadObjectsFromTgzHelmChart(fileOrDir); err != nil {
+				return nil, err
+			}
+
+			contextsByDir[fileOrDir] = ctx
+			continue
+		}
```

Comment:
Ah, sorry for missing this before. This whole block should be moved inside the `filepath.Walk` below, since we want people to just be able to run `kube-linter lint .` and crawl all the subdirectories. As written, this will only load tgz Helm charts if the user explicitly passes the path in the args.

Reply:
Hi Viswajith, thanks for noticing it. I've updated the code.